docs/cellxgene_census_schema.md

```diff
@@ -872,8 +881,185 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
+
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>array_col</td>
+    <td colspan="2" rowspan="3">As defined in CELLxGENE dataset schema</td>
+  </tr>
+  <tr>
+    <td>array_row</td>
+  </tr>
+  <tr>
+    <td>in_tissue</td>
+  </tr>
+</tbody>
+</table>
+
+#### Obs to spatial mapping –  `census_obj["census_spatial_sequencing"][organism].obs_presence_matrix` – `SOMADataFrame`
+
+It indicates the link between an observation and a scene.  Each row corresponds to an observation with the following columns:
+
+<!-- markdownlint-disable reference-links-images -->
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>soma_joinid</td>
+    <td>int</td>
+    <td>It MUST be valid <code>soma_joinid</code> from <code>census_obj["census_spatial_sequencing"][organism].obs</code>.</td>
+  </tr>
+  </tr>
+    <td>scene_id</td>
+    <td>string</td>
+    <td>It MUST be valid <code>scene_id</code> from <code>census_obj["census_spatial_sequencing"][organism].spatial</code>.</td>
+  </tr>
+  <tr>
+    <td>data</td>
+    <td>bool</td>
+    <td>It MUST be <code>True</code> if the scene contains spatial information about the oberservation, otherwise it MUST be <code>False</code>.</td>
+  </tr>
+  </tbody>
+</table>
+<!-- markdownlint-enable reference-links-images -->
+
+#### Positions array of a Scene – `census_obj["census_spatial_sequencing"][organism].spatial[scene_id].obsl["loc"]` – `PointCloudDataFrame`
+
+`scene_id` MUST correspond to the values `scene_id` in `census_obj["census_spatial_sequencing"][organism].obs_presence_matrix`.
+
+For each observation in each Scene, spatial array positions and additional positional metadata MUST be encoded as a `PointCloudDataFrame`.  Each row corresponds to an observation with the following columns:
+
+<!-- markdownlint-disable reference-links-images -->
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>x</td>
+    <td>float</td>
+    <td>It MUST be the corresponding value in the <b>first</b> column of <code>obsm["spatial"]</code>. As defined in the CELLxGENE dataset schema.</td>
+  </tr>
+  </tr>
+    <td>y</td>
+    <td>float</td>
+    <td>It MUST be the corresponding value in the <b>second</b> column of <code>obsm["spatial"]</code>. As defined in the CELLxGENE dataset schema.</td>
+  </tr>
+  <tr>
+    <td>soma_joinid</td>
+    <td>integer</td>
+    <td>It MUST be valid <code>soma_joinid</code> from <code>census_obj["census_spatial_sequencing"][organism].obs</code>.</td>
+  </tr>
+  </tr>
+  </tr>
+ </tbody>
+</table>
+<!-- markdownlint-enable reference-links-images -->
+
+If Visium ("EFO:0010961") the units for the spatial array positions are pixels from the high-resolution image (`spatial[scene_soma_joinid].img[library_id]`). Otherwise TBD.
```

Comment:
"TBD" ? I think this needs to be more specific for this version.

Reply:
It is my understanding that the x,y positions are pixels from the full resolution image.

Reply:
I would still be inclined to just not specify here, since we don't specify anything about this in the cellxgene specification. E.g. I don't think we would say it's invalid if the submitted positions had been converted to µm.

Reply:
Making sure I am understanding you correctly. Are you saying that you want to omit the entire line 1017? FYI, the X and Y coordinates are based off of the [tissue_positions.csv](https://www.10xgenomics.com/support/software/space-ranger/3.0/analysis/spatial-outputs#tissue-positions) file (which are in relation to the full res image) are used in adata.obsm['spatial'], per CxG schema 5.3. 

I'm not sure I am fully understanding what the statement is saying. Have the units for the positions been scaled to the high res image?

Reply:
Ah, no. I think we should keep what is said about Visium. I would like to not have any statement about what units are used for Slide-seq

Reply:
> Have the units for the positions been scaled to the high res image?

Ah, also no. Good catch. We store a scaling transformation from the fullres pixel space to the hires pixel space. So this should say:

```suggestion
If Visium ("EFO:0010961") the units for the spatial array positions are pixels from the full-resolution image.
```

Though we could also just say something more like: "these are the coordinates from `.obsm["spatial"]` as defined in the cellxgene schema".

Reply:
Yes, either would work. Both makes sense to me. Thanks!
On a side note, is there a particular reason the full res image is not accessible from the census?

Reply:
> On a side note, is there a particular reason the full res image is not accessible from the census?

When we were prototyping we had some issues around alignment of the full res image and the hires image not being consistent across datasets.

Reply:
Oh! I didn't realize there were alignment issues with some of the images. 
> not being consistent across datasets.

Not entirely sure I know what this means, but happy to help to see if it was a curation issue when we were wrangling the full resolution image. FWIW, there has been feedback that the full resolution image is much more useful in computational and pathology analyses. Can discuss offline if you would like.